framework/src/utils/FunctionParserUtils.C

```diff
@@ -148,11 +148,11 @@ FunctionParserUtils<is_ad>::addFParserConstants(
   unsigned int nconst = constant_expressions.size();
   if (nconst != constant_names.size())
     mooseError("The parameter vectors constant_names (size " +
-               std::to_string(constant_names.size()) + ") and constant_values (size " +
+               std::to_string(constant_names.size()) + ") and constant_expressions (size " +
```

Comment:
Why get rid of the other variable name changes?  Someone changed the name from constant_values to constant_expressions so that should be the name everywhere.

Reply:
because it does not compile right now

Reply:
constant_expressions is already used in that scope

Reply:
Thanks @GiudGiud